biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -474,34 +478,36 @@ subroutine logging_litter_fluxes(currentSite, currentPatch, newPatch, patch_site
-      real(r8) :: remainder_area      ! current patch's remaining area after donation [m2]
-      real(r8) :: leaf_m              ! leaf element mass [kg]
-      real(r8) :: fnrt_m              ! fineroot element mass [kg]
-      real(r8) :: sapw_m              ! sapwood element mass [kg]
-      real(r8) :: store_m             ! storage element mass [kg]
-      real(r8) :: struct_m            ! structure element mass [kg]
-      real(r8) :: repro_m             ! reproductive mass [kg]
-      real(r8) :: retain_frac         ! fraction of litter retained in the donor patch
-      real(r8) :: donate_frac         ! fraction of litter sent to newly formed patch
-      real(r8) :: dcmpy_frac          ! fraction going into each decomposability pool
-      integer  :: dcmpy               ! index for decomposability pools
-      integer  :: element_id          ! parteh global element index
-      integer  :: pft                 ! pft index
-      integer  :: c                   ! cwd index
-      integer  :: nlevsoil            ! number of soil layers
-      integer  :: ilyr                ! soil layer loop index
-      integer  :: el                  ! elemend loop index
+      real(r8) :: direct_dead            ! Mortality count through direct logging
+      real(r8) :: indirect_dead          ! Mortality count through: impacts, infrastructure and collateral damage
+      real(r8) :: trunk_product_site     ! flux of carbon in trunk products exported off site      [ kgC/site ] 
+                                         ! (note we are accumulating over the patch, but scale is site level)
+      real(r8) :: delta_litter_stock     ! flux of carbon in total litter flux                     [ kgC/site ]
+      real(r8) :: delta_biomass_stock    ! total flux of carbon through mortality (litter+product) [ kgC/site ]
+      real(r8) :: delta_individual       ! change in plant number through mortality [ plants/site ]
+      real(r8) :: leaf_litter            ! Leafy biomass transferred through mortality [kgC/site]
+      real(r8) :: root_litter            ! Rooty + storage biomass transferred through mort [kgC/site]
+      real(r8) :: ag_wood                ! above ground wood mass [kg]
+      real(r8) :: bg_wood                ! below ground wood mass [kg]
+      real(r8) :: remainder_area         ! current patch's remaining area after donation [m2]
+      real(r8) :: leaf_m                 ! leaf element mass [kg]
+      real(r8) :: fnrt_m                 ! fineroot element mass [kg]
+      real(r8) :: sapw_m                 ! sapwood element mass [kg]
+      real(r8) :: store_m                ! storage element mass [kg]
+      real(r8) :: struct_m               ! structure element mass [kg]
+      real(r8) :: repro_m                ! reproductive mass [kg]
+      real(r8) :: retain_frac            ! fraction of litter retained in the donor patch
+      real(r8) :: donate_frac            ! fraction of litter sent to newly formed patch
```

Comment:
It looks like `donate_frac` is calculated but no longer used for anything. delete?

Reply:
Right, I think I noticed that too.

Reply:
removed